pyhealth/data/data.py

```diff
@@ -20,9 +20,9 @@ class Event:
 
     event_type: str
     timestamp: datetime
-    attr_dict: Mapping[str, any] = field(default_factory=dict)
+    attr_dict: Mapping[str, Any] = field(default_factory=dict)
```

Comment:
Small type hint fix here, `any` is a function for iterables, and `Any` is the correct type hint.